code/clue.js

```diff
@@ -7,53 +7,220 @@ const mrGreen = {
+  occupation: 'Actress',
+  favouriteWeapon: 'Poison',
+};
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'Blue',
+  description: 'She is a Rich Socialite',
+  age: 40,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite',
+  favouriteWeapon: 'Pistol',
+};
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'Yellow',
+  description: 'He is a well decorated Army Officer',
+  age: 55,
+  image: 'assets/mustard.png',
+  occupation: 'Army Officer',
+  favouriteWeapon: 'Pistol',
+};
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'White',
+  description: 'She is a well educated woman',
+  age: 35,
+  image: 'assets/white.png',
+  occupation: 'Teacher',
+  favouriteWeapon: 'Axe',
+};
+
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
   name: 'rope',
-  weight: 10
-}
+  weight: 10,
+};
+
+const knife = {
+  name: 'Knife',
+  weight: 5,
+};
+
+const candleStick = {
+  name: 'Candle-Stick',
+  weight: 2
+};
+
+const dumbBell = {
+  name: 'Dumb-Bell',
+  weight: 20
+};
+
+const poison = {
+  name: 'Poison',
+  weight: 1
+};
+
+const axe = {
+  name: 'Axe',
+  weight: 12
+};
+
+const bat = {
+  name: 'bat',
+  weight: 12
+};
+
+const trophy = {
+  name: 'Trophy',
+  weight: 8
+};
+
+const pistol = {
+  name: 'Pistol',
+  weight: 12
+};
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const diningRoom = 'Dining Room'
+const conservatory = 'Conservatory'
+const kitchen = 'Kitchen'
+const study = 'Study'
+const library = 'Library'
+const billiardRoom = 'Billiard Room'
+const lounge = 'Lounge'
+const ballroom = 'Ballroom'
+const hall = 'Hall'
+const spa = 'Spa'
+const livingRoom = 'Living Room'
+const observatory = 'Observatory'
+const theater = 'Theater'
+const guestHouse = 'Guest House'
+const patio = 'Patio'
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+console.log(mrGreen.description)
+console.log(rope.weight)
 
+// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite,
+];
+
+const weapons = [
+  rope,
+  knife,
+  candleStick,
+  dumbBell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
 
-const weapons = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+];
 
-const rooms = []
+console.log(suspects);
+console.log(weapons);
+console.log(rooms);
+console.log()
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
```

Comment:
Looks like you've chosen to use semicolons in your code, so a little reminder to add one here. :)